### DISCUSSION TITLE
Auth  Unable to connect

### DISCUSSION THREAD

**Discussion by krahal** [supabase/supabase#18349] [Questions]
In my Dashboard, my Project Status says "Auth Unable to connect". This is preventing me from logging in or signing up users. When I call supabase.auth.signInWithOtp(), I get a 502 error. Can someone help me figure this out?

**Comment by simonhackler** [comment]
Did you manage to fix it?

**Comment by Sicria** [comment]
Having a similar issue

**Reply by kheber92** [reply]
We experience the same issue, but for one of our projects only. Other projects are working fine.

**Reply by tananga** [reply]
I have this info for 3 projects. I think that Supabase team needs to solve it. App restart in settings dont help

**Reply by GaryAustin1** [reply]
Not much info on your error but maybe this with a solution.
https://github.com/orgs/supabase/discussions/20722

**Comment by CookedApps** [comment]
Same for me. Calls to Supabase auth finish with `502 Bad Gateway`. Auth status in the dashboard is "Unable to connect" for a few days now. Tried restarting the project couple of times, didn't resolve the issue.

**Reply by tananga** [reply]
Probably something on Supabase side. Maybe @kamilogorek can solve it. He helped me last time.

**Reply by CookedApps** [reply]
Got in contact with Supabase support. They pointed out that this is an issue with a failed migration for the auth service. The temporary fix [from this issue](https://github.com/orgs/supabase/discussions/20722#discussion-6130294) helped me repair it.

**Reply by tananga** [reply]
@CookedApps Are they gonna solve it or I need to do this temporary fix?

**Reply by CookedApps** [reply]
@Tananga 
This was their reply:

> There was a migration failure that was run to upgrade your Auth service. Our Auth system is on a rapid interaction so we do run migrations to update it. For now, to unblock you we have a temporary solution suggested in this discussion by one of our Auth team members: https://github.com/orgs/supabase/discussions/20722.
> 
> We also have an ongoing task to fix this issue permanently so future migrations won't fail as well. If you're curious which migration failed on your project, it is here: https://github.com/supabase/auth/blob/aab7c3481219f136729d80d37731aa64fb8c380a/migrations/20221208132122_backfill_email_last_sign_in_at.up.sql

**Reply by tananga** [reply]
Thank you @CookedApps

**Comment by AlessandroVol23** [comment]
I've got the same issue and I'm a bit confused if this is an issue on my side or on Supabase's side?

**Reply by StunnaDawg** [reply]
Same here

**Reply by GaryAustin1** [reply]
If you are hitting the issue did you check the auth/Postgres error logs and try the migration fix if appropriate?

**Reply by StunnaDawg** [reply]
Yep! I figured it out, the migration fix was the answer. Thank you for the reply!

**Reply by AlessandroVol23** [reply]
Yep that fixed it. Super weird that some migration just breaks a lot of projects and nobody will fix it
